### DISCUSSION TITLE
Upgrading to the new version from self host

### DISCUSSION THREAD

**Discussion by MrGonzo1024** [Budibase/budibase#5303] [Help] [answered]
Hi all,

I've been using Budibase through the self-host method on DigitalOcean. I would like to upgrade to the latest release through the UI, but it's telling me that I have the latest version and it says, Current version 1.0.90. I can tell it's not the latest version because it doesn't have the map component. Do I need to manually re-install?

Thanks!

**Comment by cfepdan** [comment, marked as answer]
It may be the same issue I was having in #5276 

> Per https://github.com/Budibase/budibase/discussions/5283 watchtower container had stopped.
> docker-compose up -d watchtower-service and docker logs root_watchtower-service_1 were my friends.

**Reply by MrGonzo1024** [reply]
Thanks @cfepdan I tried that initially and it wasn't working, but when I tried again just now looks like it worked. I may not have given it enough time. Thanks again!
